tests/e2e/cairo1/test_cairo1_testing_rust.py

```diff
@@ -44,3 +44,17 @@ def test_failing_tests(protostar: ProtostarFixture, copy_fixture: CopyFixture):
         "test_panic_multiple_values: Panic [1870930782904301745253, 482670963043, 31066316372818838395891839589]"
         in result
     )
+
+
+def test_declare(protostar: ProtostarFixture, copy_fixture: CopyFixture):
```

Comment:
Naming this `test_declare` is confusing considering we have a `protostar declare` command.

Reply:
I think `test_declare_cheatcode` or something like that would be better.